fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -814,18 +808,15 @@ public void afterVisible(TransactionState txnState, boolean txnOperated) {
         if (!txnOperated) {
             return;
         }
-        executeAfterVisible(txnState);
+        executeReplayTxnAttachment(txnState);
         updateState(JobState.FINISHED);
     }
 
-    protected void executeAfterVisible(TransactionState txnState) {
-    }
-
     @Override
     public void replayOnVisible(TransactionState txnState) {
         writeLock();
         try {
-            executeReplayOnVisible(txnState);
+            executeReplayTxnAttachment(txnState);
```

Comment:
I think the name `executeReplayTxnAttachment` should be changed.
Or calling a "replay" method in "non-replay" method is weird